scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -1029,4 +1029,196 @@ class CliTest extends AbstractCliTest with CliTestBehavior {
     )
   }
 
+  test(s"handles .md files when present in includePaths") {
+    val input = string2dir(
+      s"""|/foobar.md
+        |# Hello
+        |Example usage 1 with long   spaced line
+        |```scala mdoc
+        |val  x   =   42
+        |```
+        |Example usage 2
+        |```java
+        |val  x   =   42
+        |```
+        |/.scalafmt.conf
+        |version = ${Versions.version}
+        |maxColumn   = 8
+        |project.includePaths."+" = ["glob:**.md"]
+        |""".stripMargin
+    )
+    val expected =
+      s"""|
+        |/.scalafmt.conf
+        |version = ${Versions.version}
+        |maxColumn   = 8
+        |project.includePaths."+" = ["glob:**.md"]
+        |
+        |/foobar.md
+        |# Hello
+        |Example usage 1 with long   spaced line
+        |```scala mdoc
+        |val x =
+        |  42
+        |```
+        |Example usage 2
+        |```java
+        |val  x   =   42
+        |```
+        |""".stripMargin
+
+    noArgTest(
+      input,
+      expected,
+      Seq(Array.empty[String], Array("--mode", "diff"))
+    )
+  }
+
+  test(s"ignores .md files if not present in includePaths") {
+    val input = string2dir(
+      s"""|/foobar.md
+        |# Hello
+        |Example usage 1
+        |```scala mdoc
+        |val  x   =   42
+        |```
+        |/.scalafmt.conf
+        |version = ${Versions.version}
```

Comment:
why are you specifying the version here? the versions are only used for dynamic loading of actually released versions, and there's no published version which supports this new functionality.

Reply:
Ah, I had just cargo-culted here, thinking that we always need to specify a version in the tests.
I wonder if *this* is what's killing the CI process...
If it's grabbing something that I've published locally, but is unavailable anywhere else.
Removing this.